acceptance/tests/facts/non_root_users_without_errors.rb

```diff
@@ -0,0 +1,34 @@
+test_name "C59196: running facter as a non-root user should not produce permission errors" do
+  tag 'risk:high'
+
+  # confine :except, :platform => 'aix' # bug FACT-1586
+
+  confine :except, :platform => 'windows' # this test is for unix systems
+  confine :except, :platform => 'aix' # has errors from the su command
```

Comment:
No, the system's su(1) command is printing an error message to stderr, (nothing to do with us). I'll update the comment to make that more clear